mmv1/third_party/terraform/verify/validation.go

```diff
@@ -248,6 +249,18 @@ func ValidateProjectID() schema.SchemaValidateFunc {
 	}
 }
 
+func ValidateProjectNumber() schema.SchemaValidateFunc {
```

Comment:
This function should be added via a `custom_code.constants` file as described here: https://googlecloudplatform.github.io/magic-modules/develop/custom-code/#add-reusable-variables-and-functions

Or, since this is an extremely simple regex check, you could use the `regex` options for the `item_validation` field: https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#item_validation

Reply:
Addressed, thanks Stephen!